README.md

```diff
@@ -14,8 +14,8 @@ In order to understand the library, one must first be familiar with the
 
 OpenTracing-PHP can be installed via Composer:
 
-```php
-composer require jcchavezs/opentracing-php
+```bash
+composer require opentracing/opentracing-php
```

Comment:
So that is exactly what I happens so the composer is called `jcchavezs/opentracing` but I forgot to change it here.